lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
I still believe this should be part of a separate pull request as it involves more changes.

The first step is https://github.com/indutny/bn.js/pull/81 to remove the push/unshift dependency. After that it is simple.